tests/mock.data.js

```diff
@@ -13,10 +13,17 @@ const validVc = require('./validVc.json');
 export const createRequestBody = ({issuer, vc = validVc}) => {
   const {settings: {id, options}} = issuer;
   const credential = klona(vc);
-  credential.issuer = credential?.issuer || id;
-  // convert from millisecond to seconds precision
-  // credential.issuanceDate = createISOTimeStamp();
-  // credential.issuer = id;
+  if (credential.type.includes('NoIssuer')) {
```

Comment:
However, if you're going to switch on this type value, you'll want to make sure it's defined in a context or use the examples context with the `@vocab` catch-all, so you can focus the test on the values of the `issuer.id` (or whatever else).

Reply:
I no longer use this solution as @aljones15 had a better suggestion